pkg/security/podsecuritypolicy/seccomp/types.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package seccomp
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/util/validation/field"
+)
+
+// RunAsUserStrategy defines the interface for all uid constraint strategies.
+type SeccompStrategy interface {
+	// Generate creates the profile based on policy rules.
+	Generate(pod *api.Pod, container *api.Container) (*string, error)
```

Comment:
In most of our other fields we've tried to distinguish between empty and unset so that code down the line can take actions for either case.  Maybe that doesn't apply here?